deps/chakrashim/src/inspector/v8-debugger.cc

```diff
@@ -77,22 +77,25 @@ void V8Debugger::getCompiledScripts(
     int contextGroupId,
     std::vector<std::unique_ptr<V8DebuggerScript>>& result) {
   JsValueRef scripts;
-  JsDiagGetScripts(&scripts);
+  if (JsDiagGetScripts(&scripts) != JsNoError) {
+    assert(false);
```

Comment:
@kunalspathak This was discovered due to that test case, but it won't actually fix it.  The test case needs to be marked flaky.